Update _enterprise-permissions-table.md

BrJan opened the pull request:
Fixed project permissions at the project level. Specifically, the docs show `Analyst` and `Developer` as having `projects_write`, but they only have `projects_read`

## What are you changing in this pull request and why?
<!---
Describe your changes and why you're making them. If related to an open 
issue or a pull request on dbt Core, then link to them here! 

To learn more about the writing conventions used in the dbt Labs docs, see the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md).
-->

## Checklist
<!--
Uncomment when publishing docs for a prerelease version of dbt:
- [ ] Add versioning components, as described in [Versioning Docs](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-entire-pages)
- [ ] Add a note to the prerelease version [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/docs/dbt-versions/core-upgrade)
-->
- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/static/_redirects`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

BrJan committed:
Update _enterprise-permissions-table.md

Fixed project permissions at the project level

runleonarun committed:
Merge branch 'current' into BrJan-patch-2

aaronbini commented on website/snippets/_enterprise-permissions-table.md:
```diff
@@ -84,7 +84,7 @@ The project roles enable you to work within the projects in various capacities.
 | Permissions              |   W   |         |       R        |     R     |     R     |           |             |          |                |             |     W      |        |
 | Profile                  |   W   |    R    |       W        |     R     |     R     |     R     |             |          |                |     R       |     R      |        |
-| Projects                 |   W   |    W    |       W        |     W     |     W     |     R     |      R      |          |                |     R       |     W      |        |
+| Projects                 |   W   |    R    |       R        |     R     |     R     |     R     |      R      |          |                |     R       |     R      |        |
```
in dbt-cloud code, what I see is `Admin`, `Database Admin`, `Git Admin`, and `Team Admin` all have Project write permission

mirnawong1 commented:
hi team, does this PR still need work? or is it good to merge once the conflicts are fixed?

aaronbini committed:
Merge branch 'current' of github.com:dbt-labs/docs.getdbt.com into BrJan-patch-2

aaronbini committed:
correct project write permissions

aaronbini commented:
> hey @BrJan and @aaronbini - wanted to follow up on this pr and see if it's still needed?

@mirnawong1 yes we should get this reviewed and merged

nm0s reviewed (approved):
They don't have `project_write` but they do have `project_develop`

mirnawong1 committed:
Merge branch 'current' into BrJan-patch-2

mirnawong1 commented:
> They don't have `project_write` but they do have `project_develop`

hey @nm0s thanks so much! so analyst and developer have project develop? does that mean the same thing as 'project read'? or did you mean that comment for Aaron's comment [here](https://github.com/dbt-labs/docs.getdbt.com/pull/4728#discussion_r1448186905)?

mirnawong1 committed:
Merge branch 'current' into BrJan-patch-2

mirnawong1 committed:
Merge branch 'current' into BrJan-patch-2

mirnawong1 committed:
Update _enterprise-permissions-table.md

updating per convo [here](https://dbt-labs.slack.com/archives/C02NCQ9483C/p1712854127875899?thread_ts=1712853013.373529&cid=C02NCQ9483C) w @matthewshaver who addressed my [explicit question](https://dbt-labs.slack.com/archives/C02NCQ9483C/p1712853596648079?thread_ts=1712853013.373529&cid=C02NCQ9483C)

mirnawong1 commented:
merging pr per convo[here](https://dbt-labs.slack.com/archives/C02NCQ9483C/p1712854127875899?thread_ts=1712853013.373529&cid=C02NCQ9483C) w @matthewshaver who addressed my [explicit question](https://dbt-labs.slack.com/archives/C02NCQ9483C/p1712853596648079?thread_ts=1712853013.373529&cid=C02NCQ9483C)

mirnawong1 reviewed (approved):
🚢

mirnawong1 commented:
preview is correct: https://docs-getdbt-r7ijy9nbo-dbt-labs.vercel.app/docs/cloud/manage-access/enterprise-permissions#project-permissions-for-project-roles
